aten/src/ATen/native/nested/NestedTensorBackward.cpp

```diff
@@ -13,6 +13,25 @@
 namespace at {
 namespace native {
 
+// See Note [nested tensor matmul] TODO in NestedTensorMath.cpp
+std::tuple<Tensor, Tensor> matmul_backward_nested(
```

Comment:
This has no nested-specific code. So I would suggest that we move this to `FunctionsManual.cpp` and remove the matmul_backward native function altogether.

Reply:
I think it's specific since it's for the "generalized bmm" case only.

Reply:
I think this formula would error for vector case, where we cannot `transpose(-1, -2)`

Reply:
yes it's checked by
```
TORCH_CHECK(
      self_dim >= 3,
      "matmul: For nested tensors, only inputs with >= 3 dims are currently supported. 1st input has rank: ",
      self_dim);
  TORCH_CHECK(
      mat2_dim >= 3,
      "matmul: For nested tensors, only inputs with >= 3 dims are currently supported. 2nd input has rank: ",
      mat2_dim);
```